README.md

```diff
@@ -129,6 +139,32 @@ Other common commands:
 
 For more info see [Frontend general info](docs/frontend/general-info.md)
 
+## Pulling staging data on Windows
+
+The `fab pull-staging-data` command does not currently work on Windows due to the way quotes are handled.
+
+The recommended workaround is to run the necessary commands manually. It's important to note that there are 3 categories of commands to run:
+
+1. The ones to be run on the `web` container.
+2. The ones to be run on the `db` container.
+3. The ones to be run directly on a local terminal.
+
+You will need two separate terminal instances for the `web` and local. We suggest using the VS code editor, and the terminal in it. The `db` container instructions should be done from within the Docker terminal.
```

Comment:
```suggestion
You will need two separate terminal instances for the `web` and local. We suggest using the VS code editor, and the terminal in it. The `db` container instructions should be done from within the Docker Desktop terminal because that is one of the easiest ways to activate a shell within Docker.
```